{
  "action": {
    "error": {
      "variety": [
        "Misdelivery"
      ], 
      "vector": [
        "Unknown"
      ]
    }
  }, 
  "actor": {
    "internal": {
      "motive": [
        "NA"
      ], 
      "variety": [
        "Unknown"
      ]
    }
  }, 
  "asset": {
    "assets": [
      {
        "variety": "M - Documents"
      }
    ]
  }, 
  "attribute": {
    "confidentiality": {
      "data": [
        {
          "amount": 2800, 
          "variety": "Personal"
        }
      ], 
      "data_disclosure": "Yes", 
      "data_total": 2800, 
      "data_victim": [
        "Customer"
      ], 
      "state": [
        "Unknown"
      ]
    }
  }, 
  "discovery_method": "Ext - customer", 
  "impact": {
    "overall_rating": "Unknown"
  }, 
  "incident_id": "3C890F86-8D63-41FE-83F2-B873D46E9ED1", 
  "plus": {
    "analysis_status": "Finalized", 
    "analyst": "Spitler", 
    "attribute": {
      "confidentiality": {
        "credit_monitoring": "U"
      }
    }, 
    "created": "2016-10-18T17:51:00Z", 
    "dbir_year": 2017, 
    "github": "8460", 
    "master_id": "3C890F86-8D63-41FE-83F2-B873D46E9ED1", 
    "modified": "2016-10-18T17:51:00Z", 
    "sub_source": "PHIDBR", 
    "timeline": {
      "notification": {
        "day": 15, 
        "month": 9, 
        "year": 2016
      }
    }
  }, 
  "reference": "https://www.databreaches.net/geisinger-health-plan-notifies-2800-that-processing-error-exposed-their-phi-to-others/", 
  "schema_version": "1.3", 
  "security_incident": "Confirmed", 
  "source_id": "vcdb", 
  "summary": "Misdelivery", 
  "timeline": {
    "incident": {
      "day": 30, 
      "month": 7, 
      "year": 2016
    }
  }, 
  "victim": {
    "country": [
      "US"
    ], 
    "employee_count": "1001 to 10000", 
    "industry": "524114", 
    "region": [
      "019021"
    ], 
    "state": "PA", 
    "victim_id": "Geisinger Health Plan"
  }
}